blocks/components/editable/index.js

```diff
@@ -58,6 +61,35 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onNewBlock() {
```

Comment:
When is this function called? I cannot find any documentation for TinyMCE's `NewBlock` event. Based on its name, I would have thought it'd been called on a simple enter press, but in debugging this is not the case. Can we add a DocBlock explaining when this is expected to occur.

Reply:
Having some memories about a hint from @iseulde from the TinyMCE source code. The idea was that each time a new paragraph is created the function is called or something like that :)